### PR TITLE
Feature/fsar 38 list showing

### PR DESCRIPTION
**Pull request by Yinuo-M**

### CODE REVIEW

**Comment by c-ancia** [comment]
Can you pull main on this branch please? I see the Button changes aren't synced with the amends done in the button PR. Thanks :)

**Comment by Yinuo-M** [comment]
> Can you pull main on this branch please? I see the Button changes aren't synced with the amends done in the button PR. Thanks :)
Pulled! :)

File: src/pattern-library/components/Showing/showing.html.twig [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<p class="showing">Showing
+	<span class="showing--bold">{{ displayed_results }}</span>
+	of
+	<span class="showing--bold">{{ total_results }}
+		results</span>
```
[the remark this answers is not in the document]
Comment (c-ancia) [reply src/pattern-library/components/Showing/showing.html.twig:5]: Also, is it normal results goes on a new line like this?
Maybe you could do instead
```
<span>
  {{ variable }} text
</span>
```
?

[the remark this answers is not in the document]
Comment (Yinuo-M) [reply src/pattern-library/components/Showing/showing.html.twig]: Params and newline have been fixed

**Review by PeterBerryman — APPROVED** [review]

**Pull request merged**